### ISSUE TITLE
[DOC] Add sidebar on homepage of API docs

### ISSUE THREAD

**Issue by raybellwaves** [rapidsai/docs#487]
## Report needed documentation

**Describe the documentation you'd like**

The home page of the API libraries e.g https://docs.rapids.ai/api/cudf/stable/ and https://docs.rapids.ai/api/cuml/stable/ are mostly landing pages with a summary and a list of the contents of the page. It would be nice is there was a way to configure the landing page to also have the sidebar that is present in the main documentation (see images below). I'm mostly interested a fast way to land on the installation page (https://docs.rapids.ai/install). My user journey (albeit may be unique) was https://github.com/rapidsai/cudf -> https://docs.rapids.ai/api/cudf/stable/ -> https://docs.rapids.ai/api/cudf/stable/user_guide/ then https://docs.rapids.ai/api (once I thought the installation guide would be there).

Nice work on the standardized docs.

![Screenshot 2024-03-02 at 6 17 51 PM](https://github.com/rapidsai/docs/assets/17162724/fb7b00e0-20cd-4094-9ab0-b9e09b9b95d5)
![Screenshot 2024-03-02 at 6 21 30 PM](https://github.com/rapidsai/docs/assets/17162724/529e8452-9b0c-4000-9ef8-68f35e5095e1)

**Steps taken to search for needed documentation**
List any steps you have taken:

**Comment by ajschmidt8** [comment]
This particular request will need to be handled in cudf's docs since this page is generated via Sphinx.

@bdice, should we move this issue to the cudf repo?

**Comment by bdice** [comment]
@ajschmidt8 Yes, we can add the sidebar -- but this was the original goal:

> I'm mostly interested a fast way to land on the installation page (https://docs.rapids.ai/install).

I think this would be handled by the docs/main site integration, right?

**Comment by ajschmidt8** [comment]
> I think this would be handled better by the docs/main site integration, right? The sidebar itself isn't the goal.

Yes, we'll have some navigation improvements then. This will likely be address as part of those improvements.
